plugins/modules/packaging/language/gem.py

```diff
@@ -62,6 +62,17 @@
       These gems will be independent from the global installed ones.
       Specifying this requires user_install to be false.
     required: false
+  bindir:
+    type: path
+    description:
+    - Install executables into a specific directory.
+    version_added: 3.3.0
+  norc:
+    type: bool
+    default: "no"
```

Comment:
```suggestion
    default: false
```

Reply:
The next person touch this module is probably going to be confused about the style to follow :thinking:

Reply:
At least the person will see one correct usage of `default` for booleans in this module :)

Reply:
(Feel free to create a follow-up PR which cleans up the other options. If I would have a lot more free time, I'd do that myself, but I have too many other things to work on already ;) )